.github/scripts/check-orphaned-issues.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Script to find open issues in the camunda repository that are not assigned to any project
+# This helps ensure all issues are triaged and prioritized by the relevant teams
+
+ORG_NAME="camunda"
+REPO_NAME="camunda"
+PAGE_SIZE=100
+MAX_PAGES=50  # Safety limit to avoid infinite loops
+
+echo "Checking for open issues not assigned to any project in ${ORG_NAME}/${REPO_NAME}..."
+
+# Initialize variables for pagination
+ORPHANED_ISSUES=""
+ORPHANED_COUNT=0
+CURSOR="null"
+
+for ((page=1; page<=MAX_PAGES; page++)); do
+  # Build query with pagination cursor
+  if [[ "$CURSOR" == "null" ]]; then
+    CURSOR_ARG=""
+  else
+    CURSOR_ARG=", after: \"$CURSOR\""
+  fi
+
+  # Query for open issues in the repository
+  RESPONSE=$(gh api graphql -f query="
+    query {
+      repository(owner: \"$ORG_NAME\", name: \"$REPO_NAME\") {
+        issues(states: OPEN, first: $PAGE_SIZE$CURSOR_ARG) {
+          pageInfo {
+            hasNextPage
+            endCursor
+          }
+          nodes {
+            number
+            title
+            url
+            projectItems(first: 1) {
+              totalCount
+            }
+          }
+        }
+      }
+    }")
+
+  # Extract issues that have no project assignments (totalCount = 0)
+  PAGE_ISSUES=$(echo "$RESPONSE" | jq -r '
+    .data.repository.issues.nodes[]
+    | select(.projectItems.totalCount == 0)
+    | "• <\(.url)|#\(.number)>: \(.title)"')
+
+  # Add issues from this page
+  if [[ -n "$PAGE_ISSUES" ]]; then
+    if [[ -z "$ORPHANED_ISSUES" ]]; then
+      ORPHANED_ISSUES="$PAGE_ISSUES"
+    else
+      ORPHANED_ISSUES="$ORPHANED_ISSUES"$'\n'"$PAGE_ISSUES"
+    fi
+    # Update count after adding issues
+    ORPHANED_COUNT=$(echo "$ORPHANED_ISSUES" | wc -l | tr -d ' ')
+  fi
+
+  echo "  Page $page: Found $ORPHANED_COUNT orphaned issue(s) so far..."
+
+  # Check if there are more pages
+  HAS_NEXT=$(echo "$RESPONSE" | jq -r '.data.repository.issues.pageInfo.hasNextPage')
+  if [[ "$HAS_NEXT" != "true" ]]; then
+    echo "  No more pages available"
+    break
+  fi
+
+  # Get cursor for next page
+  CURSOR=$(echo "$RESPONSE" | jq -r '.data.repository.issues.pageInfo.endCursor')
+done
+
+# Calculate final count
+if [[ -n "$ORPHANED_ISSUES" ]]; then
+  ORPHANED_COUNT=$(echo "$ORPHANED_ISSUES" | wc -l | tr -d ' ')
+fi
+
+# Output results to GITHUB_OUTPUT
+if [[ -z "$ORPHANED_ISSUES" ]]; then
+  echo "✅ No orphaned issues found!"
+  echo "orphaned-issues-found=0" >> "$GITHUB_OUTPUT"
+  echo "orphaned-issues-list=" >> "$GITHUB_OUTPUT"
+else
+  echo "⚠️  Found $ORPHANED_COUNT orphaned issue(s)"
+  
+  # Escape newlines for GitHub Actions output
+  ORPHANED_ISSUES_ESCAPED="${ORPHANED_ISSUES//$'\n'/\\n}"
+  
+  {
+    echo "orphaned-issues-found=$ORPHANED_COUNT"
+    echo "orphaned-issues-list=$ORPHANED_ISSUES_ESCAPED"
```

Comment:
Would it be possible to instead of posting the list of issues to Slack, paste a link to the Github issues page with the appropriate query parameters applied to show the issues that need to be reviewed